### ISSUE TITLE
Build failed on riscv64: cannot guess build type; you must specify one.

### ISSUE THREAD

**Issue by qyl27** [TelepathyIM/telepathy-salut#2]
I am trying to package telepathy-salut for a Linux distro on riscv64 architecture.  
Using the latest release, got a build error, configure: error: cannot guess build type; you must specify one.  
It was caused by outdated `config.guess` and `config.sub` files in the latest release.

**Comment by jbruechert** [comment]
Does `autoreconf -fi` help?

**Comment by qyl27** [comment]
Yes, but shall we have a new release? Some Linux distros were not happy with `autoreconf -fi`.  
Btw, they prefer the latest stable version, our one is 0.8.1, which has a build failed from the source.
